scripts/sct_extract_metric.py

```diff
@@ -103,18 +131,26 @@ def get_parser():
              "of consecutive labels use ':'. Example: 1:3 is equivalent to 1,2,3. Maximum Likelihood (or MAP) is "
              "computed using all tracts, but only values of the selected tracts are reported."
     )
+    optional.add_argument(
+        '-list-labels',
+        action=_ListLabelsAction,
```

Comment:
For the above suggestion.

```suggestion
        '-list-labels',
        nargs=0,
        action=_ListLabelsAction,
```

Reply:
so is the idea to accept this commit, and remove the `__init__` from above?

Reply:
Yes, exactly. (Unless there was a separate reason for using `__init__` that I missed. I had never seen that usage before so it was unclear to me.)